ibis/backends/pandas/convert.py

```diff
@@ -83,3 +83,11 @@ def convert_String(cls, s, dtype, pandas_type):
         # TODO(kszucs): should switch to the new pandas string type and convert
         # object columns using s.convert_dtypes() method
         return s.map(str, na_action="ignore").astype(object)
+
+    @classmethod
+    def convert_Struct(cls, s, dtype, pandas_type):
+        # dt.normalize returns a frozendict, which doesn't play well
+        # with the pandas executor
```

Comment:
It's just the formatter when pandas repr's a series containing a `FrozenDict`, the actual execution is fine. See #8687.

Reply:
It actually isn't, if you leave out this method the test fails. See the linked issue again, I added some clarification

Reply:
resolved in https://github.com/ibis-project/ibis/pull/8693, so this is no longer relevant.